pkgs/development/ocaml-modules/tweetnacl/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, buildDunePackage
+, ocaml
+, fetchFromGitHub
+
+, alcotest
+, bigstring
+, hex
+, ocplib-endian
+, zarith
+}:
+
+buildDunePackage {
+  pname = "tweetnacl";
+  version = "unstable-2018-04-24";
```

Comment:
Is adding a library which has been unmaintained for 3 years and has never been published to OPAM really necessary? 😐

Reply:
I just looked again at the package I'm trying to build and it seems to have integrated `tweetnacl` into its code.

Reply:
That seems better, let's drop that package then.